### ISSUE TITLE
Signing thorws "Unknown font format" Error

### ISSUE THREAD

**Issue by david-loe** [documenso/documenso#676]
### Issue Description

When signing a document including a Signature Field an Error is thrown.

### Steps to Reproduce

1. Open Gitpod Workspace (or any clean installation)
2. Sign Up
3. Upload any file (e.g. https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf)
4. Add a signer
5. Add a signature field onto the document
6. Send
7. Open `Sign Document` Link in the Email
8. Try to sign

### Unhandled Runtime Error

**Error: Unknown font format**
Call Stack
```
Object.create
../node_modules/@pdf-lib/fontkit/dist/fontkit.es.js (17839:10)
create
../node_modules/pdf-lib/es/core/embedders/CustomFontEmbedder.js (37:57)
call
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (100:22)
step
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (81:52)
next
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (74:70)
new Promise
<anonymous>
__awaiter
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (70:11)
CustomFontEmbedder.for
../node_modules/pdf-lib/es/core/embedders/CustomFontEmbedder.js (33:25)
fontkit
../node_modules/pdf-lib/es/api/PDFDocument.js (881:72)
call
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (100:22)
step
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (81:52)
next
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (74:70)
new Promise
<anonymous>
__awaiter
../node_modules/pdf-lib/node_modules/tslib/tslib.es6.js (70:11)
PDFDocument.embedFont
../node_modules/pdf-lib/es/api/PDFDocument.js (861:25)
```

## Backend Console
```
@documenso/web:dev:  ⨯ ../node_modules/@pdf-lib/fontkit/dist/fontkit.es.js (17839:10) @ Object.create
@documenso/web:dev:  ⨯ Error: Unknown font format
@documenso/web:dev:     at new Promise (<anonymous>)
@documenso/web:dev:     at new Promise (<anonymous>)
@documenso/web:dev: null
```

### Operating System [e.g., Windows 10]

Win 10

### Browser [e.g., Chrome, Firefox]

Firefox

### Version [e.g., 2.0.1]

main (c054fc78a490302c7b05224a8b459ff8c1cdc019)

### Please check the boxes that apply to this issue report.

- [X] I have searched the existing issues to make sure this is not a duplicate.
- [X] I have provided steps to reproduce the issue.
- [X] I have included relevant environment information.
- [X] I have included any relevant screenshots.
- [X] I understand that this is a voluntary contribution and that there is no guarantee of resolution.
- [ ] I want to work on creating a PR for this issue if approved

**Comment by Mythie** [comment]
Feels like Gitpod doesn’t like me, I might need to dedicate some time to coding in it so I can catch all these preemptively. Appreciate your patience with these bugs 😄 

I’ll hope to have this one sorted later tonight!

**Comment by david-loe** [comment]
Haha :D I actualy think it's not Gitpod.. ;) But it's very helpful for me to be sure that the error is not due to my dev enviroment

**Comment by crediblebilal** [comment]
@Mythie  This error occurs in Local Development setup as well.

**Comment by catalinpit** [comment]
Thanks for reporting this!

**Comment by david-loe** [comment]
Fixed by 823034911487d5785473ca8aef9bfe46e271be88
